{"name": "test_login_1", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_login_rerun.TestLogin object at 0x0379FE30>\n\n    def test_login_1(self):\n        \"\"\"记住密码登录\"\"\"\n        print(\"test_login_1记住密码登录\")\n        num = random.randint(1,2)\n        if num % 2:\n            assert True\n        else:\n>           assert False  # 断言失败\nE           assert False\n\nscripts\\test_login_rerun.py:13: AssertionError"}, "description": "记住密码登录", "start": 1602221580921, "stop": 1602221580921, "uuid": "1a56df5b-d7a5-4a6f-9c81-16de9537b1e2", "historyId": "4eba568bb2ae9c6747740f157128bdfe", "testCaseId": "94c7bce98e70b18e52ea0a4586601440", "fullName": "scripts.test_login_rerun.TestLogin#test_login_1", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_login_rerun"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-FI47AR7N"}, {"name": "thread", "value": "7328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_login_rerun"}]}